conda/recipes/dask-cudf/meta.yaml

```
@@ -26,13 +26,13 @@ requirements:
   host:
     - python
     - cudf {{ version }}
-    - dask>=2021.09.1
-    - distributed>=2021.09.1
+    - dask>=2021.09.1,<=2021.11.2
+    - distributed>=2021.09.1,<=2021.11.2
     - cudatoolkit {{ cuda_version }}
   run:
     - python
     - cudf {{ version }}
-    - dask>=2021.09.1
+    - dask>=2021.09.1,<=2021.11.2
     - distributed>=2021.09.1
```

@galipremsagar Is this supposed to be pinned as well?
```suggestion
    - distributed>=2021.09.1,<=2021.11.2
```

---

yes, thanks for catching this Bradley!